windows/code/RNDeviceInfoCPP.h

```diff
@@ -833,6 +837,25 @@ namespace winrt::RNDeviceInfoCPP
           promise.Resolve(op.GetResults());
         });
     }
+	
+	REACT_SYNC_METHOD(getHostSync);
+    std::string getHostSync() noexcept
```

Comment:
Spaces vs tabs - no auto-formatter or check on spacing on windows I guess? Trivial change though, will commit it and squash-merge to push this through

```suggestion
    REACT_SYNC_METHOD(getHostSync);
    std::string getHostSync() noexcept
```